templates/sections/event-body-two-column.html.twig

```diff
@@ -0,0 +1,18 @@
+{% set right_class = 'flex-6-of-12' %}
+{% set left_class = 'flex-6-of-12' %}
+
+<div{{ attributes.addClass('centered-container', 'su-events-details', 'flex-container', settings.extra_classes) }}>
+   <div{{ attributes.addClass('flex-8-of-12', 'flex-container') }}>
```

Comment:
@jenbreese This line shouldn't double down on the same `{{ attributes }}` variable. Please either create a new one or just set the `class` attribute on the nested div. Also, I don't think you need two flex-containers.